perun-core/src/main/java/cz/metacentrum/perun/core/entry/GroupsManagerEntry.java

```diff
@@ -244,16 +254,25 @@ public void addMember(PerunSession sess, Group group, Member member) throws Inte
 		getGroupsManagerBl().addMember(sess, group, member);
 	}
 
-	public void removeMember(PerunSession sess, Group group, Member member) throws InternalErrorException, MemberNotExistsException, NotGroupMemberException, PrivilegeException, GroupNotExistsException {
+	public void removeMember(PerunSession sess, Group group, Member member) throws InternalErrorException, MemberNotExistsException, NotGroupMemberException, PrivilegeException, GroupNotExistsException, WrongAttributeAssignmentException, AttributeNotExistsException, ExternalyManagedException {
 		Utils.checkPerunSession(sess);
 		getGroupsManagerBl().checkGroupExists(sess, group);
 		getPerunBl().getMembersManagerBl().checkMemberExists(sess, member);
+		Attribute attrSynchronizeEnabled = getPerunBl().getAttributesManagerBl().getAttribute(sess, group, GROUPSYNCHROENABLED_ATTRNAME);
+
+		if (attrSynchronizeEnabled.getValue() != null) {
+			String synchronizeEnabled = (String) attrSynchronizeEnabled.getValue();
+			// Check if group is synchronized
+			if (synchronizeEnabled.equals("true")) {
```

Comment:
This can be very nicely simplified to:
```java
if (Objects.equals("true", (String) attrSynchronizeEnabled.getValue()) {
 throw new ExternalyManagedException(...)
}
```
same for ``addMember()`` method.